src/data_provider/src/packages/pkgWrapper.h

```diff
@@ -28,7 +28,9 @@ class PKGWrapper final : public IPackageWrapper
 {
     public:
         explicit PKGWrapper(const PackageContext& ctx)
-            : m_architecture{UNKNOWN_VALUE}
+            : m_version{UNKNOWN_VALUE}
```

Comment:
Why would we have a default value just for some fields?
In `brewWrapper.h` all the fields have one.

Reply:
Added for vendor and description.